combo_cases_and_deaths/delphi_combo_cases_and_deaths/run.py

```diff
@@ -9,46 +9,68 @@
 from datetime import date, timedelta, datetime
 from itertools import product
 import re
-import sys
 
 import covidcast
 import pandas as pd
 
-from delphi_utils import read_params, create_export_csv
-from .constants import *
-from .handle_wip_signal import *
+from delphi_utils import read_params
+from .constants import METRICS, SMOOTH_TYPES, SENSORS, GEO_RESOLUTIONS
+from .handle_wip_signal import add_prefix
 
 
 def check_not_none(data_frame, label, date_range):
     """Exit gracefully if a data frame we attempted to retrieve is empty"""
     if data_frame is None:
         print(f"{label} not available in range {date_range}")
-        sys.exit(1)
+        return False
+    return True
 
-def combine_usafacts_and_jhu(signal, geo, date_range):
+def maybe_append(df1, df2):
+    """
+    If both data frames are available, append them and return. Otherwise, return
+    whichever frame is not None.
+    """
+    if df1 is None:
+        return df2
+    if df2 is None:
+        return df1
+    return df1.append(df2)
+
+COLUMN_MAPPING = {"time_value": "timestamp",
+                  "geo_value": "geo_id",
+                  "value": "val",
+                  "stderr": "se",
+                  "sample_size": "sample_size"}
+def combine_usafacts_and_jhu(signal, geo, date_range, fetcher=covidcast.signal):
     """
     Add rows for PR from JHU signals to USA-FACTS signals
     """
-    usafacts_df = covidcast.signal("usa-facts", signal, date_range[0], date_range[1], geo)
-    jhu_df = covidcast.signal("jhu-csse", signal, date_range[0], date_range[1], geo)
-    check_not_none(usafacts_df, "USA-FACTS", date_range)
-    check_not_none(jhu_df, "JHU", date_range)
+    print("Fetching usa-facts...")
+    usafacts_df = fetcher("usa-facts", signal, date_range[0], date_range[1], geo)
+    print("Fetching jhu-csse...")
+    jhu_df = fetcher("jhu-csse", signal, date_range[0], date_range[1], geo)
+
+    if (not check_not_none(usafacts_df, "USA-FACTS", date_range)) and \
```

Comment:
I would change the polarity here (return True if it is None) since the double negative is hard to parse.